docs/meta/goals.rst

```diff
@@ -137,7 +137,7 @@ a standardized container image which is then used by different execution
 engines to manage application instances.
 
 However, classical configuration management still has its place. In a cloud
-environment which primarly uses containers as application delivery method,
+environment which primary uses containers as application delivery method,
```

Comment:
I think that this should stay [primarly](https://www.merriam-webster.com/dictionary/primarily) since [primary](https://www.merriam-webster.com/dictionary/primary) means something else.